pkgs/applications/editors/emacs-modes/melpa-packages.nix

```diff
@@ -132,9 +132,24 @@ let
         flycheck-rtags = fix-rtags super.flycheck-rtags;
 
         pdf-tools = super.pdf-tools.overrideAttrs (old: {
-          nativeBuildInputs = [ pkgs.pkg-config ];
-          buildInputs = with pkgs; old.buildInputs ++ [ autoconf automake libpng zlib poppler ];
-          preBuild = "make server/epdfinfo";
+          nativeBuildInputs = [
+            pkgs.autoconf
+            pkgs.automake
+            pkgs.pkg-config
+            pkgs.removeReferencesTo
+          ];
+          buildInputs = old.buildInputs ++ [ pkgs.libpng pkgs.zlib pkgs.poppler ];
+          preBuild = ''
+            make server/epdfinfo
+            remove-references-to \
+              -t ${pkgs.stdenv.cc.libc.dev} \
```

Comment:
@adisbladis This seems to break on darwin with `attribute 'dev' missing` on latest `nixpkgs-unstable`.

Should this one be optional? Is there a similar but different dev to remove references to on darwin?